docs/rules/call-super-in-init.md

```diff
@@ -0,0 +1,24 @@
+## Call _super in init lifecycle hooks
+
+### Rule name: `call-super-in-init`
```

Comment:
I'd call this rule `require-super-in-init`, it's common way of naming eslint rules when you require something to be present. wdyt?